packages/components/Organisms/Markup/Markup.vue

```diff
@@ -125,11 +125,27 @@ $responsive-iframe-padding-top: $rpl-space-4 !default;
       @include rpl_text_color($rpl-markup-link-color);
     }
   }
+
   /* Iframes */
   &__iframe-container {
-    @include rpl_responsive_iframe;
```

Comment:
safe to delete the scss file if no longer used?

Reply:
> safe to delete the scss file if no longer used?

Thanks @rubyllamadora , redundant css now is removed.